### DISCUSSION TITLE
Collapse on the ToC does not work below the first level

### DISCUSSION THREAD

**Discussion by alemsabic** [CONNECT-platform/codedoc#100] [Q&A]
In the documentation it's written: "As mentioned above, Collapse can be used in a nested manner, so you can have your desired level of depth in your ToC". 

When I use the example that follows ...

[Home](/)

> :Collapse label=Stuff
>
> [Overview](/some-folder/overview)
> > :Collapse label=Spec
> > [Spec Overview](/some-folder/spec/overview)
> > [Spec of This](/some-folder/spec/this)

... this is the result:

![Screenshot at Feb 12 17-39-50](https://user-images.githubusercontent.com/9151275/107795791-67726800-6d59-11eb-9e45-8099b0d02c7c.png)

**Comment by loreanvictor** [comment]
For any [quoted component](https://codedoc.cc/docs/markdown/overview#quoted-components) (including `Collapse`), you need [a newline between the first line (which specifies the component type and its parameters) and the content](https://codedoc.cc/docs/markdown/overview#code1-l4). In your case, this is your markdown, where the newline is missing:

```md
[Home](/)

> :Collapse label=Stuff
>
> [Overview](/some-folder/overview)
> > :Collapse label=Spec
> > [Spec Overview](/some-folder/spec/overview)
> > [Spec of This](/some-folder/spec/this)
```

And this is the correct version:

```md
[Home](/)

> :Collapse label=Stuff
>
> [Overview](/some-folder/overview)
> > :Collapse label=Spec
> >
> > [Spec Overview](/some-folder/spec/overview)
> > [Spec of This](/some-folder/spec/this)
```

**Comment by loreanvictor** [comment]
@alemsabic if this 👆fixes your issue, please tell me so that I would convert this issue to a discussion thread under Q&A.

**Comment by alemsabic** [comment]
@loreanvictor Hi Eugene, yeah, it does. Thanks for the fast answer. So it's an error in the documentation. 
![Screenshot at Feb 12 18-30-15](https://user-images.githubusercontent.com/9151275/107801592-8e806800-6d60-11eb-97b3-d39d17ada012.png)

**Comment by loreanvictor** [comment]
> So it's an error in the documentation.

Yes it is, thanks for mentioning.
